src/pages/login.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from 'next/head';
+import { FC } from 'react';
+
+import LoginForm from '@components/molecules/LoginForm';
+import AuthLayout from '@src/layouts/auth';
+
+const Login: FC = () => {
+  return (
+    <AuthLayout>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Billy.io</title>
+      </Head>
+      <LoginForm type="login" />
```

Comment:
With the enum change:
```suggestion
      <LoginForm type={EAuthType.Login} />
```